presto-google-sheets/src/main/java/io/prestosql/plugin/google/sheets/SheetsMetadata.java

```diff
@@ -148,6 +161,55 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         return schemaTableNames;
     }
 
+    @Override
+    public ConnectorInsertTableHandle beginInsert(ConnectorSession session, ConnectorTableHandle tableHandle)
+    {
+        SheetsTableHandle sheetsTableHandle = (SheetsTableHandle) tableHandle;
+        return new SheetsInsertTableHandle(sheetsTableHandle.getSchemaName(),
+                sheetsTableHandle.getTableName(), buildColumnHandles(sheetsTableHandle));
+    }
+
+    @Override
+    public Optional<ConnectorOutputMetadata> finishInsert(ConnectorSession session, ConnectorInsertTableHandle insertHandle, Collection<Slice> fragments, Collection<ComputedStatistics> computedStatistics)
+    {
+        return Optional.empty();
+    }
+
+    @Override
+    public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean ignoreExisting)
+    {
+        List<SheetsColumnHandle> columns = buildColumnHandles(tableMetadata);
+        sheetsClient.createSheet(tableMetadata.getTable().getTableName(), columns);
+    }
+
+    @Override
+    public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorNewTableLayout> layout)
+    {
+        List<SheetsColumnHandle> columns = buildColumnHandles(tableMetadata);
```

Comment:
these two lines are same in `createTable`. can you call `createTable` from here?